packages/block-editor/src/components/block-canvas/style.scss

```diff
@@ -0,0 +1,6 @@
+iframe[name="editor-canvas"] {
+	width: 100%;
+	height: 100%;
+	background-color: $white;
```

Comment:
Do you know if the issue is present in the site editor as well? I know there were a lot of back and forth about the default background there due to the loading mechanics and whether this change could impact it. cc @jasmussen

Reply:
Thanks for checking.

Yes, for empty theme (or any theme with no set background color) the editor will have a default background color of `background-color: $gray-900;` coming from https://github.com/WordPress/gutenberg/blob/81852a55e700afdaaca7657a2382dd38c0e52b5a/packages/edit-post/src/components/visual-editor/style.scss#L14

Reply:
> Do you know if the issue is present in the site editor as well?

This issue has also occurred in the Site Editor in the past, but it has been fixed in #46314. After that, the selector was changed in part of #46752.

There is also an issue where the background color flashes white when the Site Editor loads for the first time (#57001). This problem occurs both in trunk and in this PR, so I think it will need to be fixed in a follow-up.

Reply:
Oh, thanks for correcting me @t-hamano 

I meant to say, "without a fallback like this..." 😄 

This PR unifies those previous fixes for the site editor with the post editor so they'll work the same, running from the same CSS rule.

Reply:
CC: @jameskoster — you were there for some of these conversations. Do you recall the details?